openedx/adg/lms/applications/views.py

```diff
@@ -291,13 +291,13 @@ def get_context_data(self, request):
         return context
 
 
-class EducationAndExperienceView(RedirectToLoginOrRelevantPageMixin, TemplateView):
+class MyExperienceView(RedirectToLoginOrRelevantPageMixin, TemplateView):
```

Comment:
In my opinion, EducationAndExperience completely explains all that the view entails. 
The view or page basically contains Education, Work Experience, Background Question (about experience) and References (about experience and education). Therefore, we should not rename this.

Secondly, when we are merging the two tickets, this might create a hassle.

Reply:
@ahmed-faraz46 the reason behind not changing the name of the view and templates ^